docker-compose.yml

```diff
@@ -40,6 +40,8 @@ services:
       - /app/node_modules
     environment:
       - BACKEND_API_URL=${BACKEND_API_URL-}
+      - PRODBLOOMURL = ${PROD_BLOOM_URL}
```

Comment:
it should just be one BLOOM_URL
that we configure differently for the two deployments

Reply:
Yes, it was by mistake added and forgot to clear it up. Thanks